ubiquibot-config-default.json

```diff
@@ -51,9 +51,39 @@
       "weight": 5
     }
   ],
-  "default-labels": ["Time: <1 Hour", "Priority: 0 (Normal)", "Test"],
-  "auto-pay-mode": true,
-  "disable-analytics": true,
+  "command-settings": [
+    {
+      "name": "start",
+      "enabled": false
+    },
+    {
+      "name": "stop",
+      "enabled": false
+    },
+    {
+      "name": "wallet",
+      "enabled": false
+    },
+    {
+      "name": "payout",
+      "enabled": false
+    },
+    {
+      "name": "multiplier",
+      "enabled": false
+    },
+    {
+      "name": "query",
+      "enabled": false
+    },
+    {
+      "name": "allow",
+      "enabled": false
+    }
+  ],
+  "default-labels": [],
+  "assistive-pricing": false,
+  "auto-pay-mode": false,
   "comment-incentives": false,
   "max-concurrent-assigns": 2,
```

Comment:
I just realized that perhaps this should be a value that represents no limit. Technically because this isn't a JavaScript file, we can't use `Infinity` but some substitute would make sense.

Reply:
What [properties](https://github.com/devpanther/bounty-bot/blob/350f404e8d98f080176079122c3e2246fdf72df8/ubiquibot-config-default.json) exactly should allow the `Infinity` word? I don't see any where `Infinity` is applicable.

Reply:
`max-concurrent-assigns: Infinity`

Reply:
I don't really like it because under the hood in the js code we would need to transform this `Infinity` to either `undefined` either `Number.MAX_SAFE_INTEGER` which adds unnecessary code complexity.

Reply:
We should do something or set it as `undefined` to represent no limits though.

Reply:
If we set `max-concurrent-assigns` to `undefined` then at some point in the code we would need to write smth like:
```
const bountyHunterMax = getBountyHunterMaxFromConfig() ?? Number.MAX_SAFE_INTEGER;
```

Perhaps [here](https://github.com/ubiquity/ubiquibot/blob/development/ubiquibot-config-default.json#L89) we can simply set the default value for the `max-concurrent-assigns` parameter to be `Number.MAX_SAFE_INTEGER (9007199254740991)`

This way there are (almost) no limits for max number of assigned issues unless a partner project sets its own value for the `max-concurrent-assigns` parameter

Reply:
Sure thing! We should also include a comment that explains why we used that integer.